packages/@okta/vuepress-site/guides/sign-into-mobile-app/section09/index.md

```diff
@@ -5,6 +5,6 @@ title: Next Steps
 
 Now that you can sign users into your mobile app, you may need additional information on other topics to help with other tasks.
 
-[Protect an API Guide]
-[Customization Guide]
+[Protect an API Guide]<br>
```

Comment:
Another of those "should this be a list" places.  I won't call out the others, but wanted to give more than one example